dts/bindings/sensor/zephyr,senss-phy-sensor.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023, Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: Sensor subsystem (senss) physical sensor properties bindings.
+
+include: zephyr,senss-sensor.yaml
+
+properties:
+  underlying-device:
+    type: phandle
+    required: true
+    description: underlying sensor device for physical sensor
```

Comment:
Base on Sensing subsystem implementation, we have to add a device node for each physical sensor. 
So that Sensing subsystem can handle these physical sensors as  common sensor instances of sensing subsystem.